src/Datalust.SerilogMiddlewareExample/Diagnostics/SerilogMiddleware.cs

```diff
@@ -8,49 +8,50 @@
 
 namespace Datalust.SerilogMiddlewareExample.Diagnostics
 {
-    class SerilogMiddleware
+    public class SerilogMiddleware
     {
-        const string MessageTemplate = "HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms";
+        private const string MessageTemplate = "HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms";
 
-        static readonly ILogger Log = Serilog.Log.ForContext<SerilogMiddleware>();
+        private static readonly ILogger Log = Serilog.Log.ForContext<SerilogMiddleware>();
 
-        readonly RequestDelegate _next;
+        private readonly RequestDelegate _next;
 
         public SerilogMiddleware(RequestDelegate next)
         {
-            if (next == null) throw new ArgumentNullException(nameof(next));
-            _next = next;
+            _next = next ?? throw new ArgumentNullException(nameof(next));
         }
 
         public async Task Invoke(HttpContext httpContext)
         {
             if (httpContext == null) throw new ArgumentNullException(nameof(httpContext));
 
-            var start = Stopwatch.GetTimestamp();
+            var sw = Stopwatch.StartNew();
             try
             {
                 await _next(httpContext);
-                var elapsedMs = GetElapsedMilliseconds(start, Stopwatch.GetTimestamp());
+                sw.Stop();
```

Comment:
Could you please limit the change here to only the path-related logic? The formatting changes make it difficult to spot issues like this one, which would be a (slight) regression.

Reply:
done